x-pack/plugins/cases/public/components/case_view/components/case_view_activity.test.tsx

```diff
@@ -496,54 +496,6 @@ describe('Case View Page activity tab', () => {
       });
     });
 
-    // FLAKY: https://github.com/elastic/kibana/issues/152206
-    describe.skip('Reporter', () => {
```

Comment:
Thinking about adding a test to make sure CaseViewActivity calls `parseCaseUsers` with the right params but then I will have to mock its return value which is a bit complex 😬